ref-arch/real-time-analytics.rst

```diff
@@ -0,0 +1,509 @@
+.. _introduction:
+
+Real Time Analytics
+#####################
+
+Over the last few years we've helped many different kinds of clients use Citus and noticed
+a technical problem many businesses have: running real-time analytics over large streams
+of data.
+
+For example, say you're building an HTTP analytics dashboard. You have enough clients that
+around every millisecond a user hits one of their websites and sends a log record to you.
```

Comment:
The word client has a specific meaning in an HTTP context. Maybe customers?

e.g.

You have enough clients that around every millisecond a user hits one of their websites and sends a log record to you.

->

Overall, requests from your customers generate around 1000 log records per second.